src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceStringParts = sourceString.split(';');
+  const strWithoutSpaces = sourceStringParts.map((str) => str.trim());
+  const result = strWithoutSpaces.reduce((prev, el) => {
```

Comment:
This is not the `previous` value. This is a `accumulator`. Change it.
Also u can return it immediately)

```suggestion
  return strWithoutSpaces.reduce((acc, el) => {
```